### PR TITLE
User profile, activity, avatar

### PR DESCRIPTION
**Pull request by danielmdavis**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
           <li><%= link_to 'Admin', users_path, :class => "test" %></li>
           <% end %>
           <li><%= link_to '+ Add New Spot', new_spot_path, :class => "test" %></li>
-          <li><%= link_to 'My Activity' %></li>
+          <li><%= link_to 'Activity' %></li>
```
Comment (ChoiChips) [inline app/views/layouts/application.html.erb]: "Activity should be "My Activity"

File: app/uploaders/avatar_uploader.rb [hunk 2]
```diff
@@ -14,7 +14,12 @@ class AvatarUploader < CarrierWave::Uploader::Base
   # Override the directory where uploaded files will be stored.
   # This is a sensible default for uploaders that are meant to be mounted:
   def store_dir
-    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+    # "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```
Comment (AppDevEvan) [inline app/uploaders/avatar_uploader.rb]: Remove commented out code

File: app/uploaders/avatar_uploader.rb [hunk 3]
```diff
@@ -39,12 +44,12 @@ def store_dir
 
   # Add a white list of extensions which are allowed to be uploaded.
   # For images you might use something like this:
-  # def extension_whitelist
-  #   %w(jpg jpeg gif png)
+  def extension_whitelist
+    %w(jpg jpeg gif png)
+  end
+  # def content_type_whitelist
```
Comment (AppDevEvan) [inline app/uploaders/avatar_uploader.rb]: Commented out code - Pls remove

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -16,3 +16,52 @@
...
+  width: 280px;
+  background-color: #c0fefe;
+  text-align: center;
+  font-family: helvetica, 'sans serif';
+  overflow: scroll;
+  font-size: .8em;
+}
+.show {
+  margin: 20px 0px;
+  /* padding: 8px 30px; */
```
Comment (ChoiChips) [inline app/assets/stylesheets/application.css]: Remove comment

**Pull request merged**